client/web/compose/src/components/Common/RecordToolbar.vue

```diff
@@ -27,6 +27,29 @@
         </b-button>
       </div>
 
+      <b-button-group class="ml-auto">
+        <b-button
+          v-if="nextPrevNavigationOptions.prev"
+          pill
+          size="lg"
+          variant="outline-primary"
+          class="mr-2"
+          @click="$emit('update-navigation:prev')"
```

Comment:
Make sure button is disabled if there is no previous/next record.
Also a :title will be required since the button is icon only